src/js/espn-borischentiers.js

```diff
@@ -58,15 +90,27 @@ function setTierInfo(pos) {
   });
 }
 
+function removeTierHeader() {
+  var tierHeader = document.getElementById(TIER_HEADER_ID);
+  tierHeader.parentNode.removeChild(tierHeader);
+}
+
 /* Add header cell */
 function addTierHeader() {
   const headerRow = document.getElementsByClassName(TABLE_HEADER_CLASS)[0].children[0];
   let newTh = document.createElement('th');
   newTh.innerHTML = COLUMN_NAME;
-  newTh.rowSpan = 2;
+  newTh.id = TIER_HEADER_ID;
+  newTh.colSpan = 2;
```

Comment:
Why are we changing `rowSpan` to `colSpan` here? The purpose of it was so the box would take up 2 rows as the header is 2 rows (because of the header and sub-header).

Reply:
Ah, that makes sense. I was just inspecting ESPN's html and saw that all of the headers had colspan defined on them.